app/Components/Search.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import { checkDatabaseForSearch } from './Helpers/HomeHelper'
+
+
+
+class Search extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      search: '',
+      scrubbedTweets: { "contentItems": []},
+    }
+  }
+
+  helper() {
+    checkDatabaseForSearch(this.state.search, this.setState.bind(this), this.state.scrubbedTweets)
+    this.props.history.replace('/Home')
+  }
+
+
+  render () {
+    return (
+      <article className='home-search'>
+        {/* {this.displayErrorMessage()} */}
```

Comment:
It's a flash message that existed in the component I took it out of. I'm keeping it for when I bring that method over later 👍🏻